source/agora/node/Ledger.d

```diff
@@ -229,14 +239,63 @@ public class Ledger
         else
             return null;
     }
+
+    /***************************************************************************
+
+        Write a block to the ledger file on disk.
+
+        Params:
+            fd = Instance of LedgerFile
+            block = Type of struct to writeFile
+
+    ***************************************************************************/
+
+    public void writeBlockFile (File file, ref Block block) @trusted
```

Comment:
Also the `Block` parameter should be made `const`

Reply:
added